### ISSUE TITLE
Cannot import or run initial configuration on fresh install

### ISSUE THREAD

**Issue by JoelMiller74** [sqlcollaborative/dbaclone#101]
I believe that the most common way and best practice to install a module is using the command:
`Install-Module DBAClone`
After installing the module, the most common way to start using that module is to import that module using the name:
`Import-Module DBAClone`
This is the process I am using.

However, when importing the module using this process, we are presented with the following error:
`WARNING: [09:49:29][Get-DcnClone] The module setup has NOT yet successfully run. Please run 'Set-DcnConfiguration'`
When trying to use that command, you are presented with the following error:
`WARNING: [09:35:38][Get-DcnClone] The module setup has NOT yet successfully run. Please run 'Set-DcnConfiguration'
Set-DcnConfiguration: The 'Set-DcnConfiguration' command was found in the module 'dbaclone', but the module could not be loaded. For more information, run 'Import-Module dbaclone'.`

I cannot import the module, and I cannot run the Set-DcnConfiguration command.

This was not the behavior when originally using PSDatabaseClone and it is not the behavior laid out in the current readme of DBAClone. In fact, if I were to install and import PSDatabaseClone it would automatically have me configure the module and it would work. Since it was configured, I could then UPGRADE to DBAClone and it would still work as it was already configured.
When following the instructions in the current Readme, it references that we should import the module using the path and references the old PSDatabaseClone syntax:
`Import-Module .\PSDatabaseClone`
It then states 'Setup for the module has not yet run. Starting.' and starts the configuration process.
However, when importing DBAClone, this is not the experience.

I am using version 0.10.12 of DBAClone.
I have tried on Windows Powershell 5.1 and Powershell 7.
I have tried on Windows 10, Windows Server 2016 and 2019.
I have run Powershell within a normal and elevated session.

I have almost never submitted an issue on Github before so if I missed anything, came across poorly, or there is anything additional I can try or provide, please let me know.

![DBAClone Import Issue](https://user-images.githubusercontent.com/32984717/85147453-80e80200-b214-11ea-896a-9a66609d405e.png)

**Comment by sanderstad** [comment]
Thank you for creating this issue.

At some point PSDatabaseClone was renamed to dbaclone. This was to keep the naming convention the same as other tools within the SqlCollaborative.

For some reason the command that tests the configuration was referencing to the old configurations for PSDatabaseClone. 
I have fixed this and it's being tested as we speak

**Comment by sanderstad** [comment]
Please run the following to get the latest update
```powershell
Update-Module dbaclone
```

**Comment by JoelMiller74** [comment]
Thanks! I appreciate the quick update and that makes sense. However, I updated the module to 0.10.14 and still receive the same responses from `import-module dbaclone' and `Set-DcnConfiguration`2. Any ideas?

**Comment by sanderstad** [comment]
You have to run the Set-DcnConfiguration once. The configurations used with PSDatabaseClone will not work and they need to be reset with that command
When you run `Set-DcnConfiguration` it will ask for a couple of settings and it should then work

**Comment by JoelMiller74** [comment]
I cannot run a `Set-DcnConfiguration` configuration command in DBAClone until the module is imported. I cannot import it as it has not been configured:
`[07:36:16][228.75 ms] C:\git> Set-DcnConfiguration`
`WARNING: [08:09:22][Get-DcnClone] The module setup has NOT yet successfully run. Please run 'Set-DcnConfiguration'`
`Set-DcnConfiguration: The 'Set-DcnConfiguration' command was found in the module 'dbaclone', but the module could not be loaded. For more information, run 'Import-Module dbaclone'.`
`[08:09:24][1.31 s] C:\git> Import-Module dbaclone`
`WARNING: [08:12:05][Get-DcnClone] The module setup has NOT yet successfully run. Please run 'Set-DcnConfiguration'`

The older PSDatabaseClone module used to allow you to `import-module PSDatabaseClone` and it would see that you have not configured the product and automatically start the internal configuration process for you

That is also how the readme of DBAClone shows how to use the product. If you import the module and it is not configured, it will start the configuration process for you automatically.

![image](https://user-images.githubusercontent.com/32984717/85408113-b862f100-b529-11ea-9fe5-8c19f4e47a7a.png)

**Comment by sanderstad** [comment]
I need to update those images but I have not yet had the chance.

Did you also download the repository, because I see a `Git` folder.
Do you have dbaclone installed in multiple locations? Maybe some sort of older version gets loaded instead of the latest one?

Maybe remove all the installed versions of dbaclone and install it from scratch

**Comment by delapohl** [comment]
I'm running into the same issue with a fresh install of dbaclone 0.10.16:
![image](https://user-images.githubusercontent.com/4336138/93373213-09316f80-f823-11ea-826c-b21092aa732e.png)

I can't seem to proceed with import or configuration of the module.

**Comment by sanderstad** [comment]
@delapohl @JoelMiller74 
I have finally been able to reproduce this problem.
To find it I created a completely fresh Windows 10 VM and did a fresh install on the machine. This caused me to get the same error as you did.

The problem appears to happen because of the compiling that is being done in one of the builds. I'm not 100% sure why this was the problem but I fixed the problem. In version 0.10.23 this problem should not occur.

![image](https://user-images.githubusercontent.com/6154981/95454034-2cf05d00-096c-11eb-95b2-33b6688b36c1.png)
